src/main/java/com/android/dexdeps/DexDataException.java

```diff
@@ -20,4 +20,5 @@
  * Bad data found inside a DEX file.
  */
 public class DexDataException extends RuntimeException {
+    private static final long serialVersionUID = 1L;
```

Comment:
Possible to disable warnings for the Java sources?  They're all sourced from AOSP, and I try to touch them as little as possible (with one notable exception).